docs/content/operator/gas-station/deployment/deployment.mdx

```diff
@@ -55,7 +55,7 @@ git clone https://github.com/iotaledger/gas-station
 
 ```
 cd gas-station/docker
-../utils/./gas-station-tool.sh generate-sample-config --config-path config.yaml --docker-compose -e testnet
+ENV=testnet ../utils/./gas-station-tool.sh generate-sample-config --config-path config.yaml --docker-compose
```

Comment:
```suggestion
../utils/./gas-station-tool.sh generate-sample-config --config-path config.yaml --docker-compose
```

Actually an outdated command. Should be this now looking at the tool.
You can specify the network with the `network` flag